frontend/projects/upgrade/src/app/app.module.ts

```diff
@@ -21,14 +21,15 @@ export const getEnvironmentConfig = (http: HttpClient, env: Environment) => {
   }
 
   // in a prod build, we currently need to fetch environment.json at runtime
-  // to provide apiBaseUrl and googleClientId
+  // to provide apiBaseUr, googleClientId, featureFlagNavToggle and withinSubjectExperimentSupportToggle
```

Comment:
I don't see the feature flag toggle. Are we adding that later?

Reply:
Actually featureFlagNavToggle is merged. But just checked, its in dev and not in release/v5.1 & it missed this comment. Probably we can add the comment later in dev.